js/script.js

```diff
@@ -0,0 +1,79 @@
+var books = [];
+
+const form = document.querySelector('.book-form');
+const formId = document.querySelector('.book-form__input--id');
+const formTitle = document.querySelector('.book-form__input--title');
+const formEditionYear = document.querySelector('.book-form__input--year');
+const formAuthor = document.querySelector('.book-form__input--author');
+const formCost = document.querySelector('.book-form__input--cost');
+const formNumberOfCopies = document.querySelector('.book-form__input--copies');

```

Comment:
good choice to select by class , however it might return multiple values for each element. One style class can be applied to multiple elements. How we can avoid this ? We need some attribute of the element that is unique across  entire document . Can you improve the approach of selecting elements ?